runtime/src/accounts_background_service.rs

```diff
@@ -89,11 +89,13 @@ impl SnapshotRequestHandler {
         accounts_db_caching_enabled: bool,
         test_hash_calculation: bool,
         use_index_hash_calculation: bool,
+        non_snapshot_time_us: u128,
     ) -> Option<u64> {
         self.snapshot_request_receiver
             .try_iter()
             .last()
             .map(|snapshot_request| {
+                let mut total_time = Measure::start("wallclock time elapsed");
```

Comment:
would be nice if this mentions something about snapshots. With nvtx labels "wallclock time elapsed" won't be very descriptive.

Reply:
will handle in separate pr